File: programers_Lv1/문자열을 정수로 바꾸기.py
# https://school.programmers.co.kr/learn/courses/30/lessons/12925
def solution(s):
    s_str = str(s)
    prefix = s_str[0] if s_str[0] == "+" or s_str[0] == "-" else ''
    value = s_str[1:] if s_str[0] == "+" or s_str[0] == "-" else s_str[:]
    return int(prefix + value)

print(solution(-1234))

File: programers_Lv1/문자열을 정수로 바꾸기.js
// https://school.programmers.co.kr/learn/courses/30/lessons/12925?language=javascript
function solution(s) {
  var prefix =
    s.slice(0, 1) === "+" || s.slice(0, 1) === "-" ? s.slice(0, 1) : "";
  var value = s.slice(0, 1) === "+" || s.slice(0, 1) === "-" ? s.slice(1) : s;

  return ~~(prefix + value);
}

console.log(solution("-1234"));
